core/modules/filter/filter.admin.inc

```diff
@@ -543,6 +544,18 @@ function filter_editor_image_upload_settings_form($format) {
   foreach ($stream_wrappers as $scheme => $info) {
     $options[$scheme] = $info['description'];
   }
+  
+  $form['orientate'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable image re-orientation'),
+    '#description' => t('If EXIF data indicates the need, the image will be rotated appropriately before being saved.'),
+    '#default_value' => $settings['orientate'],
+    '#attributes' => array(
+      'data-editor-image-rotate' => 'orientate',
+    ),
+    '#states' => $show_if_image_uploads_enabled,
```

Comment:
We should put `#access => function_exists('exif_read_data')` here, so we don't show an option that won't work.

Reply:
Done.